modules/ztp-image-based-upgrade-concept.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-concept_{context}"]
+= Using the `ImageBasedGroupUpgrade` CR for the image-based upgrade with {ztp}
```

Comment:
IBGU is not ztp or gitops specific. The emphasis here should be the `Group` concept. And we should make it clear that it's a CR on the hub.